content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -625,6 +626,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `MixedProtocolLBService`: Enable using different protocols in the same `LoadBalancer` type
   Service instance.
 - `MultiCIDRRangeAllocator`: Enables the MultiCIDR range allocator.
+- `MultiCIDRServiceAllocator`: Enables a new IPAddress object kind, and a new Service ClusterIP allocator.
+  The new allocator removes previous Service CIDR block size limitations for IPv4, and limits IPv6 size to a /64.
```

Comment:
```suggestion
- `MultiCIDRServiceAllocator`: Track IP address allocations (for example, for Service cluster IPs) 
  using IPAddress objects.
```
Further details are best mentioned in https://kubernetes.io/docs/reference/networking/virtual-ips/#something (the text above can link there)

Reply:
this allocator only works for Services